subsys/lorawan/Kconfig

```diff
@@ -44,35 +44,57 @@ config LORAWAN_PUBLIC_NETWORK
 	  Enable this option to use a public LoRaWAN network.
 	  Disable for private LoRaWAN networks.
 
-config LORAMAC_REGION_AS923
-	bool "Asia 923MHz Frequency band"
+choice
+	prompt "LoRaWAN Region Select"
+	default LORAMAC_REGION_AS923
+	config LORAMAC_REGION_AS923
+		bool "Asia 923MHz Frequency band"
 
-config LORAMAC_REGION_AU915
-	bool "Australia 915MHz Frequency band"
+	config LORAMAC_REGION_AU915
+		bool "Australia 915MHz Frequency band"
 
-config LORAMAC_REGION_CN470
-	bool "China 470MHz Frequency band"
+	config LORAMAC_REGION_CN470
+		bool "China 470MHz Frequency band"
 
-config LORAMAC_REGION_CN779
-	bool "China 779MHz Frequency band"
+	config LORAMAC_REGION_CN779
+		bool "China 779MHz Frequency band"
 
-config LORAMAC_REGION_EU433
-	bool "Europe 433MHz Frequency band"
+	config LORAMAC_REGION_EU433
+		bool "Europe 433MHz Frequency band"
 
-config LORAMAC_REGION_EU868
-	bool "Europe 868MHz Frequency band"
+	config LORAMAC_REGION_EU868
+		bool "Europe 868MHz Frequency band"
 
-config LORAMAC_REGION_KR920
-	bool "South Korea 920MHz Frequency band"
+	config LORAMAC_REGION_KR920
+		bool "South Korea 920MHz Frequency band"
 
-config LORAMAC_REGION_IN865
-	bool "India 865MHz Frequency band"
+	config LORAMAC_REGION_IN865
+		bool "India 865MHz Frequency band"
 
-config LORAMAC_REGION_US915
-	bool "North America 915MHz Frequency band"
+	config LORAMAC_REGION_US915
+		bool "North America 915MHz Frequency band"
 
-config LORAMAC_REGION_RU864
-	bool "Russia 864MHz Frequency band"
+	config LORAMAC_REGION_RU864
+		bool "Russia 864MHz Frequency band"
+
+endchoice
+
+config LORAWAN_CHANNELS_MASK_SIZE
```

Comment:
This should be a enum in the lorawan header, not a kconfig value.
Multiple regions can be enabled, which this does not express well.

Reply:
Fixed.